utils/kata-manager.sh

```diff
@@ -261,6 +269,7 @@ Options:
                 https://containerd.io/releases/#support-horizon
  -d           : Enable debug for all components.
  -D           : Install Docker server and CLI tooling (takes priority over '-c').
+ -N           : Install nerdctl (takes priority over '-c', only implemented for x86_64 and ARM).
```

Comment:
Could you add an explicit arch check and error if we don't find one of these architectures?